.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[**]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+max_line_length = 80
+
+# Set default charset and indentation for python files
+[{*.py,coalib/**/*.py,./coala,tests/**/*.py}]
+charset = utf-8
```

Comment:
If we have `charset = utf-8` in `[**]` block, then no need to add it in `[{*.py,coalib/**/*.py,./coala,tests/**/*.py}]` block.